Document/0x05b-Basic-Security_Testing.md

```diff
@@ -105,9 +105,30 @@ An extensive list of root detection methods is presented in the "Testing Anti-Re
 
 For a typical mobile app security build, you'll usually want to test a debug build with root detection disabled. If such a build is not available for testing, you can disable root detection in a variety of ways that will be introduced later in this book.
 
-#### Recommended Tools
-
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1227>
+#### Recommended Tools - Android device
+
+There are many tools and frameworks for security testing Android applications used throughout this guide. In the next sections you will learn more about some of their main commands and interesting use cases. Please check the official documentation for installation instructions of the following tools/apks:
+
+- APK Extractor: App to extract apks without root.
+- frida server: Server for Frida, the dynamic instrumentation toolkit for developers, reverse-engineers, and security researchers.
+- objection apk: APK for objection, the runtime mobile exploration toolkit, powered by Frida. Useful with non-rooted devices.
```

Comment:
objection -> Objection 
Objection should only be for the host computer, not the android device

Reply:
Please also do uppercase for all other tools listed below